examples/tensorflow_distributed/tensorflow_distributed.py

```diff
@@ -0,0 +1,78 @@
+# A basic example showing how to use Runhouse to run a TensorFlow distributed training script on a cluster of GPUs
+
+import asyncio
+import json
+import os
+
+import runhouse as rh
+import tensorflow as tf
+
+
+# Define the Tensorflow distributed training script
+def train():
+    # Initialize the distributed training environment,
+    # per https://www.tensorflow.org/tutorials/distribute/multi_worker_with_keras
+    tf_config = json.loads(os.environ["TF_CONFIG"])
+    strategy = tf.distribute.MultiWorkerMirroredStrategy()
+    num_workers = strategy.num_replicas_in_sync
+    print(f"Worker {tf_config['task']['index']} of {num_workers} initialized")
+
+    # Create a simple model and optimizer
+    model = tf.keras.Sequential([tf.keras.layers.Dense(10, activation="relu")])
+    optimizer = tf.keras.optimizers.SGD(0.01)
+
+    with strategy.scope():
+        model.compile(optimizer=optimizer, loss="mse")
+
+    model.fit(
+        tf.data.Dataset.from_tensor_slices(
+            (tf.random.normal([1000, 10]), tf.random.normal([1000, 1]))
+        ).batch(32)
+    )
+
+    print(f"Worker {tf_config['task']['index']} finished")
+
+
+async def main():
+    # Create a cluster of 2 GPUs
+    gpus_per_node = 1
+    num_nodes = 2
+    cluster = (
+        rh.cluster(
+            name=f"rh-{num_nodes}x{gpus_per_node}GPU",
+            instance_type=f"A10G:{gpus_per_node}",
+            num_instances=num_nodes,
+        )
+        .up_if_not()
+        .save()
```

Comment:
Is this implying they should be using Den? Maybe remove `.save()` (in both examples) unless I have that wrong or we want to encourage it